e2e/tests/plugins/telemetryTable/telemetryTable.e2e.spec.js

```diff
@@ -0,0 +1,101 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2022, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+const { test, expect } = require('@playwright/test');
```

Comment:
@ozyx i haven't had time to finish the documentation, but we should update new templates to use the fixtures.js file to pick up some additional base functionality in the test suites

Reply:
Updated to use the augmented `test` object from `fixtures.js`. Actually, it causes the test to fail since it's now picking up some console errors that were being thrown from the `mct-tree` Vue component. 

I'm also noticing that some of our other e2e tests that create new domain objects also fail due to this same error:

```
[error] [Vue warn]: Error in nextTick: \"TypeError: Cannot read properties of undefined (reading 'scrollTop')\"
    
    found in
    
    ---> <MctTree> at src/ui/layout/mct-tree.vue
           <FormControlComponent> at src/api/forms/components/controls/Locator.vue
             <Root>,[error] TypeError: Cannot read properties of undefined (reading 'scrollTop')
        at VueComponent.isItemInView (webpack-internal:///./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/ui/layout/mct-tree.vue?vue&type=script&lang=js&:490:53)
        at VueComponent.eval (webpack-internal:///./node_modules/vue-loader/lib/index.js??vue-loader-options!./src/ui/layout/mct-tree.vue?vue&type=script&lang=js&:476:31)
        at Array.eval (webpack-internal:///./node_modules/vue/dist/vue.js:1997:14)
        at flushCallbacks (webpack-internal:///./node_modules/vue/dist/vue.js:1923:16),
```

In this particular case, it seems to be thrown when clicking "CREATE" and then immediately clicking "OK". So far I've tried some different wait options to no avail.

Reply:
Clarification to the above, the tests are failing on my local instance. So maybe a non-issue(?).

The tests are passing on CI.

Reply:
@ozyx I believe the issue is related to this chrome bug https://github.com/nasa/openmct/issues/5184#issuecomment-1125408027

The way I worked around it was by selecting the destination folder https://github.com/nasa/openmct/blob/03e7d912bea4d83540b4751b3eee5ecba56c2931/e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js#L45-L46

Reply:
It definitely seems related to the Chromium issue. BTW, I just realized that CI is actually not running this test yet since it's not included in the npm script, so scratch what I said about it passing on CI. 🤦‍♂️

Should I go ahead and add `telemetryTable` to the npm script?

Reply:
@ozyx Before adding new tests to every commit, I've let the new tests run for a few nights (all e2e tests are run during the nightlies) to see if they're flaky on CI Agents (timing issues are really hard to find on local runs from my experience).